examples/user_guide/03-Applying_Customization.ipynb

```diff
@@ -24,9 +24,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As introduced in the [Customization](../getting_started/2-Customization.ipynb) section of the 'Getting Started' guide, HoloViews separates the notion of content from presentation. This is achieved by tracking a set of keyword values that specify how elements are to appear. These keywords can be specified for a given type and can be overridden by their ``group`` and ``label`` specification (as introduced in [Annotating Data](../user_guide/01-Annotating_Data.ipynb)) or set for individual element instances.\n",
+    "As introduced in the [Customization](../getting_started/2-Customization.ipynb) section of the 'Getting Started' guide, HoloViews maintains a strict separation between your content (your data and declarations about your data) and its presentation (the details of how this data is represented visually). This separation is achieved by maintaining sets of keyword values (\"options\") that specify how elements are to appear, stored outside of the element itself. Option keywords can be specified for individual element instances, for all elements of a particular type, or for arbitrary user-defined sets of plots that you give a certain ``group`` and ``label`` (see [Annotating Data](../user_guide/01-Annotating_Data.ipynb)).\n",
```

Comment:
```suggestion
     "As introduced in the [Customization](../getting_started/2-Customization.ipynb) section of the 'Getting Started' guide, HoloViews maintains a strict separation between your content (your data and declarations about your data) and its presentation (the details of how this data is represented visually). This separation is achieved by maintaining sets of keyword values (\"options\") that specify how elements are to appear, stored outside of the element itself. Option keywords can be specified for individual element instances, for all elements of a particular type, or for arbitrary user-defined sets of visualizations that you give a certain ``group`` and ``label`` (see [Annotating Data](../user_guide/01-Annotating_Data.ipynb)).\n",
```

Reply:
I don't like the word 'plots'. HoloViews isn't a plotting library! Call them 'visualizations' instead.

Reply:
Seems like it should actually be objects, not visualizations, as objects (elements) are what's given groups and labels. I'll delete this suggestion and make it objects instead.